rules/aircraft.yaml

```diff
@@ -121,32 +121,41 @@ zep:
 	AnnounceOnBuild:
 	Voiced:
 		VoiceSet: KirovVoice
-#	SpawnActorOnDeath:
-#		Actor: zephusk
+	SpawnActorOnDeath:
+		Actor: zephusk
 	Targetable@ground:
 		TargetTypes: Ground
 		UpgradeTypes: airborne
 		UpgradeMaxEnabledLevel: 0
 
-#zephusk:
-#	Inherits: ^HelicopterHusk
-#	Tooltip:
-#		Name: Kirov Airship
-#	RenderSprites:
-#		Image: zep
-#	RenderVoxels:
-#		Image: zep
-#	WithVoxelBody:
-#	WithShadow:
-#	Helicopter:
-#		ROT: 10
-#		Speed: 50
-#		AirborneUpgrades: airborne
-#	RevealsShroud:
-#		Range: 8c0
-#		Type: CenterPosition
-#	FallsToEarth:
-#		Velocity: 200
+zephusk:
+	Tooltip:
+		Name: Kirov Airship
+	Armor:
+		Type: Light
+	Health:
+		HP: 140
+	Aircraft:
+		TurnSpeed: 10
+		Speed: 50
+		AirborneUpgrades: airborne
+		CanHover: True
+		Voice: Move
+	FallsToEarth:
+		Spins: False
```

Comment:
I left a comment although to me it doesn't make any sense. They aren't helicopters with rear rotors.